vue/components/ui/organisms/form/form.vue

```diff
@@ -27,16 +27,34 @@
                                     v-bind:key="field.value"
                                 >
                                     <input-ripe
-                                        v-bind:type="field.type"
+                                        v-bind:type="field.meta"
                                         v-bind="field.props"
                                         v-bind:value="values[field.value]"
                                         v-if="
-                                            ['string', 'number', 'email', 'input'].includes(
-                                                field.type
-                                            )
+                                            field.type === 'text' &&

```

Comment:
Let's simplify this by using v-else-if and placing the more specific inputs first.